pkg/orderers/pbftgoodcase.go

```diff
@@ -119,8 +119,9 @@ func (orderer *Orderer) applyCertReady(cert *availabilitypb.Cert) (*events.Event
 // propose assumes that the state of the PBFT Orderer allows sending a new proposal
 // and does not perform any checks in this regard.
 func (orderer *Orderer) propose(cert *availabilitypb.Cert) (*events.EventList, error) {
+	//TODO Alejandro: argument must be slice of pointers to certs
```

Comment:
I don't quite understand... The argument still seems to be a single certificate.

Reply:
The proto message `availabilitypb.proto:Cert` is a message class that contains a `oneof` field. Inside the `oneof` is where one can see that the proto message `Cert` is actually a `mscpb.Certs`. I decided not to change that abstraction of the `availabilitypb.proto:Cert` (for ISS, it does not matter what form, number or type a cert takes, it just needs to receive one from the availability module).